src/styles/variables.scss

```diff
@@ -13,13 +8,35 @@ $color-red: #f95252;
 $color-white: #ffffff;
 $color-yellow: #e5c650;
 
-$size-xsmall: 11px;
+$size-xxsmall: 8px;
+$size-xsmall: 12px;
 $size-small: 16px;
-$size-base: 23px;
-$size-large: 32px;
-$size-xlarge: 45px;
-$size-xxlarge: 64px;
+$size-base: 24px;
+$size-large: 40px;
+$size-xlarge: 64px;
+$size-xxlarge: 84px;
 
 $max-tablet-size: 768px;
 
-$grid-size: 992px;
\ No newline at end of file
+$grid-size: 992px;
+
+$Logo-font-size-small: 12px;
+$Logo-padding: 2px;
```

Comment:
These should go into the logo component.

Reply:
There is no Logo component, I will make one then